### ISSUE TITLE
Localization infrastructure

### ISSUE THREAD

**Issue by sandysage** [mozilla-lockbox/lockbox-android#46]
Some notes (private) from meeting with localization team:
https://docs.google.com/document/d/1Ek3pxyL_VBa2Amia8rz-qeAaMnUs2CCYL4P2kiWa-pc/edit#

Best practices:
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Land_good_quality_strings

Meeting scheduled for November 30

Depends on #204

**Comment by devinreams** [comment]
Duplicate of #519 

Nothing actionable here from us any longer, but on my list to help support and monitor the l10n team's work to hook up to their infrastructure/repo.
